lookout/style/typos/analyzer.py

```diff
@@ -284,20 +336,37 @@ def reconstruct_identifier(tokenizer: TokenParser, pred_tokens: List[str], ident
         return "".join(res)
 
     @classmethod
+    @with_uasts_and_contents(unicode=True)
```

Comment:
Should be False. Not all languages can be correctly transformed now. 
```suggestion
    @with_uasts_and_contents(unicode=False)
```